challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -2237,7 +2237,7 @@
         {
           "text": "The color stop at 0 pixels should be <code>yellow</code>.",
           "testString":
-            "assert(code.match(/yellow\\s+?0px/gi), 'The color stop at 0 pixels should be <code>yellow</code>.');"
+            "assert(code.match(/yellow\\s+?0(px)?Z/gi), 'The color stop at 0 pixels should be <code>yellow</code>.');"
```

Comment:
I believe this line should be:

`"assert(code.match(/yellow\\s+?0(px)?/gi), 'The color stop at 0 pixels should be <code>yellow</code>.');"`